src/App.js

```diff
@@ -19,7 +17,9 @@ function App() {
 
         <Routes>
           <Route path="/book/:id" element={<BookPage />} />
-          <Route path="/" element={<HomePage claimedUrl={claimedUrl} />} />
+          <Route path="/" element={<HomePage claimedUrl={0} />} />
+          <Route path="/available/" element={<HomePage claimedUrl={0} />} />
```

Comment:
I wouldn't bother with this route, just have home as unclaimed and /claimed as claimed (make sure to update the nav accordingly)